Cargo.toml

```diff
@@ -1,26 +1,23 @@
 [package]
-name = "tree-sitter-ql-dbscheme"
-description = "ql dbscheme grammar for the tree-sitter parsing library"
+name = "tree-sitter-dbscheme"
```

Comment:
Not sure about this rename. The repo is still called `tree-sitter-ql-dbscheme`, after all.

Reply:
But the parser is. Having the repo named different from the language is not really intended.

Reply:
That is because the name in `gammar.js` is wrong. The generate script is copy-pasting from there.   
I'm fixing it.

Reply:
I'm not sure I follow. The parser is _what_?

Currently, the name of the parser is `tree-sitter-ql-dbscheme`. This is also how the (to my knowledge) only downstream user [refers to it](https://github.com/github/codeql/blob/main/ql/extractor/Cargo.toml#L12).

To be clear, I would be happy to see this renamed to `tree-sitter-dbscheme`. The original name contained `-ql-` out of an abundance of caution, in case some other user of `dbscheme` was out there.

Reply:
Currently the grammar is named `dbscheme`. 
I cannot change it to `ql-dbscheme`, because then tree-sitter gives me the following error: 
`Error: Grammar's 'name' property must not start with a digit and cannot contain non-word characters.`. 

What can we do?

Reply:
Try underscore instead? `ql_dbscheme`

Reply:
Underscore seem to work

Reply:
Yep, a number of other grammars use that trick.